### ISSUE TITLE
Failures on [co19] Roll co19 to 67300c027e768d6ae000076e733c8bec1c0e7fd2

### ISSUE THREAD

**Issue by athomas** [dart-lang/co19#969]
There are a bit more failures than I expected from the commit messages, I will wait with landing the roll until you've had look.

There are new test failures on [[co19] Roll co19 to 67300c027e768d6ae000076e733c8bec1c0e7fd2](https://dart-review.googlesource.com/c/sdk/+/173720).

The tests
```
co19/LanguageFeatures/Subtyping/static/tests/left_bottom_A01_t02 CompileTimeError (expected Pass)
co19/LanguageFeatures/Subtyping/static/tests/left_bottom_A01_t04 CompileTimeError (expected Pass)
co19/LanguageFeatures/Subtyping/static/tests/left_bottom_A01_t06 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/null_check_operator_A01_t02 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/null_check_operator_A03_t01 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/null_check_operator_A05_t01 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/null_check_operator_A07_t01 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/static_errors_A21_t02 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/Least-greatest-closure/nonfunction_dynamic_t01 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/Least-greatest-closure/nonfunction_static_t01 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/assignability_A01_t01 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/assignability_A01_t02 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t01 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t02 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t03 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t04 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t05 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t06 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t07 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t08 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/exports_A01_typedef2_t09 MissingCompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/extension_method_resolution_A01_t05 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/extension_method_resolution_A01_t06 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/generics_A01_t03 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/generics_A01_t04 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/legacy_libraries_A01_t03 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/legacy_libraries_A01_t04 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/legacy_libraries_A03_t02 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/null_aware_operator_A01_t02 CompileTimeError (expected Pass)
co19/LanguageFeatures/nnbd/type-aliases/null_aware_operator_A02_t02 CompileTimeError (expected Pass)
    and 69 more tests
```
are failing on configurations
```
analyzer-asserts-strong-linux
analyzer-asserts-weak-linux
cfe-strong-linux
cfe-weak-linux
dart2js-hostasserts-strong-linux-x64-chrome
dart2js-hostasserts-weak-linux-x64-chrome
dartdevk-strong-linux-release-chrome
dartdevk-weak-linux-release-chrome
dartk-strong-linux-debug-x64
dartk-strong-linux-release-x64
dartk-strong-win-release-x64
dartk-weak-asserts-linux-debug-x64
dartk-weak-asserts-linux-release-x64
dartk-weak-asserts-win-release-x64
dartkp-strong-linux-release-x64
dartkp-strong-win-release-x64
dartkp-weak-asserts-linux-release-x64
dartkp-weak-asserts-win-release-x64
```

**Comment by sgrekhov** [comment]
@iarkh looks like 67300c027e768d6ae000076e733c8bec1c0e7fd2 wrongly removed  `// SharedOptions=--enable-experiment=nonfunction-type-aliases` from nnbd/type-aliases tests
It removed string `// SharedOptions=--enable-experiment=non-nullable,nonfunction-type-aliases` at all while the correct change was to remove  `non-nullable,` only

**Comment by sgrekhov** [comment]
Fixed by 7a43aa1.
